python/packages/ai/teams/ai/prompts/completion_config.py

```diff
@@ -87,4 +90,5 @@ def from_dict(cls, data: dict) -> "CompletionConfig":
             stop_sequences=data.get("stop_sequences"),
             temperature=data.get("temperature", 0),
             top_p=data.get("top_p", 0),
+            data_sources=data.get("data_sources"),
```

Comment:
this may cause a KeyError if the "data_sources" key is not present, to avoid that provide a default value of None like is done on line 92

Reply:
resolved